Makefile

```diff
@@ -35,6 +35,7 @@ clean:
 
 dkms: dkms.in
 	sed -e 's/#RELEASE_STRING#/$(RELEASE_STRING)/' $^ > $@
+	chmod +x $@
```

Comment:
The execute bit is already set by the install command further down. Plus last time I've looked using in-tree dkms was causing various issues.

Reply:
IMO in-tree dkms should work fine, otherwise its some bug to be fixed.

Reply:
I would love it if using dkms in-tree "just works". Although currently we `make install` before doing any tests, plus there were cases where it relied on particular files on the rootfs ... IIRC it was `/var/lib/dkms/version` or alike.

Fixing this would be amazing, although it's orthogonal to the rest of this PR so let's keep it separate. Off the top of my head we want to:
   - add the execute bit (as you've done above)
   - remove the `make install` from the CI
   - ensure dkms is not installed in the CI containers, and `/etc/dkms/` && `/var/lib/dkms/` are missing
   - fix any test fallouts
   
 Thanks in advance

Reply:
Split and addressed in separate PR. Essentially the `pwd` manipulation was removed.